stackle_app/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "stackleapp",
   "private": true,
-  "scripts" :{
-    "start" : "http-server -p 8081"
+  "scripts": {
+    "start": "gulp serve"
```

Comment:
Using Gulp to serve the app is a nice option when in Development, specially with LiveReload feature. If you are planning to use this for Production also, check how efficient this option is with lot of traffic.

Reply:
I will look in to that as well.